### ISSUE TITLE
how to use a bridge (obfs4 or snowflake) when config tor

### ISSUE THREAD

**Issue by kitoytimmy** [joelverhagen/TorSharp#84]
how to use a bridge (obfs4 or snowflake) when config tor

**Comment by joelverhagen** [comment]
Hey @kitoytimmy, I don't know much about Tor bridges, but perhaps look at this issue and the linked PR? Another person was interested in the same thing and it appears that they are not unblocked: https://github.com/joelverhagen/TorSharp/issues/48

**Comment by joelverhagen** [comment]
Closing since this appears as a duplicate of #48. Feel free to open a new issue if you see a feature gap.
